src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -59,23 +59,47 @@
                         </Polygon>
                     </Button.Resources>
                 </Button>
-            </StackPanel>
-            <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Grid.Column="1">
                 <ComboBox ItemsSource="{Binding Source={StaticResource engineEnum}, Mode=OneWay}"
                           SelectedItem="{Binding nodeModel.Engine, Mode=TwoWay, RelativeSource={RelativeSource AncestorType=dww:ModelessChildWindow}}"
                           Style="{StaticResource ResourceKey=SComboBox}"
                           Name="EngineSelectorComboBox"
                           VerticalAlignment="Bottom"
-                          HorizontalAlignment="Right"
                           VerticalContentAlignment="Center"
                           HorizontalContentAlignment="Left"
                           FontSize="14px"
                           Height="40"
                           Width="Auto"
                           Margin="1"
-                          Grid.Column="1"
-                          Foreground="{StaticResource DynamoStandardLableTextBrush}" 
-                          Visibility="Collapsed"/>
+                          Foreground="{StaticResource DynamoStandardLableTextBrush}"
+                          Visibility="Collapsed" />
+                <Button Content="Migration Assistant"
+                        Style="{DynamicResource ResourceKey=STextButtonWithShapeIcon}"
+                        Name="MigrationAssistantBtn"
+                        Click="OnMigrationAssistantClicked"
+                        VerticalAlignment="Bottom"
+                        Margin="1"
+                        ToolTip="{x:Static p:Resources.PythonScriptEditorMigrationAssistantButtonTooltip}" />
+            </StackPanel>
+            <StackPanel Orientation="Horizontal"
+                        HorizontalAlignment="Right"
+                        Grid.Column="1">
+                <Button Content="{x:Static p:Resources.PythonScriptEditorMoreInfoButton}"
+                        Style="{DynamicResource ResourceKey=STextButtonWithShapeIcon}"
+                        Name="MoreInfoButton"
+                        Click="OnMoreInfoClicked"
+                        HorizontalAlignment="Right"
+                        VerticalAlignment="Bottom"
+                        Margin="1"
+                        Grid.Column="1"
+                        ToolTip="{x:Static p:Resources.PythonScriptEditorMoreInfoButtonTooltip}">
+                    <Button.Resources>
+                        <fa:ImageAwesome x:Key="Shape"
+                                         Icon="ExclamationCircle"
```

Comment:
should be question mark ?

Reply:
yea havent changed the existing `ScriptEditorWindow` on this PR, except for adding the Migration Assistant btn